manifests/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metdata:
+  name: rukpak
```

Comment:
Would it make more sense to have manifests -- other than the CRDs -- for the specific provisioner implementations themselves; e.g.

```yaml
kind: Namespace
metadata:
    name: k8s-provisioner
```

Reply:
Yeah, I think that's a good point but we could also just a quick refactoring of the current structure further down the line when we start trying to bridge out to other provisioner implementation - I'm fine with updating though. Maybe moving them to a nested directory under the ./provisioner directory?

Reply:
@njhale Ended up updating and moved the k8s provisioner-specific YAML resources to ./provisioner/k8s/manifests.